java/blocks/jolokia-install-rhel.dck

```diff
@@ -0,0 +1,5 @@
+ADD jolokia-opts /opt/jolokia/
+ADD jolokia.properties /opt/jolokia/
+ADD https://origin-repository.jboss.org/nexus/content/groups/ea/org/jolokia/jolokia-agent-jvm/${JOLOKIA_VERSION}/jolokia-agent-jvm-${JOLOKIA_VERSION}-agent.jar /opt/jolokia/jolokia.jar
```

Comment:
Could be combined to a single block with a parameter (the base).

Also the referenced files (jolokia-opts and jolokia.properties) are not part of the block, which makes the block not self contained.

Also what I could offer is to adapt https://github.com/fabric8io/agent-bond/blob/master/fish-pepper/jolokia/install.dck to take the Maven repository as a property (if this helps).

If you dont mind I would rework the blocks myself, have to refresh my memories, though ;-)

@nicolaferraro that's probably your change, if you dont mind I would review everything in this combined PR.